common/src/main/java/org/jboss/hal/testsuite/fragment/shared/modal/WizardWindow.java

```diff
@@ -28,24 +28,57 @@ public void next() {
     }
 
     /**
-     * Clicks either Finish or Save button and wait's unit the wizard window is closed
+     * Clicks Save button and waits unit the wizard window is closed.
+     *
+     * @return true if window is not present after the button was clicked
+     */
+    public boolean save() {
+        clickSave();
+        return isWizardClosed();
+    }
+
+    /**
+     * Clicks Save button, dismiss 'Reload required' window if appears and waits unit the wizard window is closed.
+     * @return true if window is not present after the button was clicked
+     */
+    public boolean saveAndDismissReloadRequiredWindow() {
+        clickSave();
+        dismissRealoadRequitedWindow();
```

Comment:
Noticed now typo in the method name `dismissRealoadRequitedWindow` => `dismissReloadRequiredWindow`. The method name should be refactored :-)